docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+---
+version: "2.1"
+services:
+  server:
+    build:
+      context: .
+      dockerfile: ./docker/server/Dockerfile
+      args:
+        - VERSION=1.1.0
```

Comment:
🎏 Should `VERSION` be set here?

Reply:
Right now it's being used to ensure I copy the right artifact out of the target directory. I think I could avoid setting it, as long as we run a `mvn clean` or whatever the equivalent is.